jellyfin_apiclient_python/configuration.py

```diff
@@ -24,9 +24,12 @@ def __init__(self):
 
         LOG.debug("Configuration initializing...")
         self.data = {}
+
         self.http()
+        self.app()
+        # self.auth()
 
-    def app(self, name, version, device_name, device_id, capabilities=None, device_pixel_ratio=None):
+    def app(self, name=None, version=None, device_name="Firefox", device_id=None, capabilities=None, device_pixel_ratio=None):
```

Comment:
```
name=None, version=None, device_name="Firefox", device_id=None
```
I do know that some functions of the Jellyfin API refuse to work without a correct device name or id. It has also been known to have websocket connectivity break when the client name is changed.